midi_app_controller/gui/utils.py

```diff
@@ -141,8 +141,8 @@ def get_selected_action_id(self) -> Optional[str]:
 
 def is_subpath(path: Path, subpath: Path) -> bool:
     """Checks if one path represents a file/directory inside the other directory."""
-    path_str = str(subpath.resolve().absolute())
-    subpath_str = str(path.resolve().absolute())
+    path_str = str(path.resolve().absolute())
+    subpath_str = str(subpath.resolve().absolute())
     return path_str.startswith(subpath_str)
```

Comment:
Everywhere it is used, the `subpath` is the first argument. So they need to be switched here.